data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -50,12 +51,16 @@ from dimcand
         select distinct on (cand_sk) * from dimcandproperties
             order by cand_sk, candproperties_sk desc
     ) cand_p_most_recent using(cand_sk, cand_id)
-    left join dimcandoffice co on co.cand_sk = dimcand.cand_sk
+    left join dimcandoffice co using (cand_sk)
     left join dimoffice using (office_sk)
     inner join dimparty using (party_sk)
+    left join dimcandproperties dcp using (cand_sk)
+    where dcp.election_yr >= :START_YEAR
```

Comment:
We want start year - 1 because we are looking at election year and if there was a special election in the first year of the cycle this wouldn't catch it.